### ISSUE TITLE
Cannot create a virtual server that differs in only the source address

### ISSUE THREAD

**Issue by girtslinde** [F5Networks/terraform-provider-bigip#317]
When I try to create two virtual servers like this:
```
resource "bigip_ltm_virtual_server" "server1" {
  name = "/Common/server1"
  destination = "2.3.4.5"
  port = 0
}

resource "bigip_ltm_virtual_server" "server2" {
  name = "/Common/server2"
  destination = "2.3.4.5"
  port = 0
  source = "3.4.5.6/32"
  
  # a dependency to repro the problem deterministically
  depends_on = [bigip_ltm_virtual_server.server1] 
}
```
I get an error:
```
Error: HTTP 400 :: {"code":400,"message":"01070333:3: Virtual Server /Common/server2 illegally shares destination address, source address, service port, ip-protocol, and vlan with Virtual Server /Common/server1.","errorStack":[],"apiError":3}
```
The problem seems to be that CreateVirtualServer() creates server2 without the source address, and then the source address gets updated after that. Though, the creation already causes a conflict with the server1. I verified that filling in the source address in CreateVirtualServer() solves the problem.

**Comment by focrensh** [comment]
Tracking with TER-425

**Comment by girtslinde** [comment]
I just noticed the same problem with the ip_protocol field as well. In fact, of the 5 fields in the error (destination address, source address, service port, ip-protocol, and vlan) CreateVirtualServer() uses only destination address and service port to create the server, so there is a potential for problems with the other 3 fields.

**Comment by RavinderReddyF5** [comment]
Fixed in BIG-IP Provider version v1.3.3, please re open if you see issue again.
